src/components/PostDetails/PostDetails.tsx

```diff
@@ -1,45 +1,86 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { NewCommentForm } from '../NewCommentForm';
 import './PostDetails.scss';
+import { getPostComments, deleteComment } from '../../api/posts';
+import { Post, NewComment } from '../../react-app-env';
 
-export const PostDetails: React.FC = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+interface Props {
+  selectedPost: Post,
+  selectedPostId: number,
+}
+
+export const PostDetails: React.FC<Props> = ({
+  selectedPost,
+  selectedPostId,
+}) => {
+  const [comments, setComments] = useState<NewComment[]>([]);
+  const [commentsVisible, setCommentsVisible] = useState(true);
+
+  useEffect(() => {
+    getPostComments(selectedPostId)
```

Comment:
you should also create getPostDetails function, and call it here for getting info about selected post